js/build_map.js

```diff
@@ -90,11 +90,19 @@ var tooltipDiv = d3.select("body").append("div")
   .classed("tooltip hidden", true);
 
 // Read data and add to map
-d3.queue()
-  .defer(d3.json, "data/state_boundaries_USA.json")
-  .defer(d3.tsv, "data/county_centroids_wu.tsv")
-  .defer(d3.json, "data/wu_data_15_range.json")
-  .await(fillMap);
+if(waterUseViz.interactionMode === 'tap') {
```

Comment:
I think there's a cleaner way to show this queue by having shared files not in the if statements. I'm submitting a PR soon, so I'll do that with that PR.